addons/mail/static/src/core/common/message.js

```diff
@@ -213,7 +214,8 @@ export class Message extends Component {
 
     get authorAvatarUrl() {
         if (
-            this.message.type === "email" &&
+            this.message.type &&
+            this.message.type.includes("email") &&
```

Comment:
not using `?.`?

Reply:
Because server data can have `false`, so `false?.` would crash